app/models/user_access.rb

```diff
@@ -84,7 +84,7 @@ def accessible_facility_groups(action)
   end
 
   def accessible_facilities(action)
-    return Facility.all.includes(:region) if power_user?
+    return Facility.all.includes(:region, facility_group: :organization) if power_user?
```

Comment:
Ohh, we were failing to run this `includes` when it mattered the most. Ouch. Nice find.

Reply:
✨

Reply:
this is something that is possible (but annoying) to add specs for to prevent regressions for this sort of perf critical includes